components/color_input.tsx

```diff
@@ -45,18 +38,17 @@ class ColorInput extends React.Component {
         }
     }
 
-    checkClick = (e) => {
-        const colorPickerDOMNode = ReactDom.findDOMNode(this.colorPicker);
-        if (!colorPickerDOMNode || !colorPickerDOMNode.contains(e.target)) {
+    private checkClick = (e: MouseEvent): void => {
```

Comment:
`React.MouseEvent<HTMLElement>` is often used in favor of `MouseEvent`. In my experience, using `React.MouseEvent<HTMLElement>` usually results in less casting when accessing properties of the event.

Thoughts @jespino?

Reply:
In this case we are not talking about a `React.MouseEvent`, it is just a `MouseEvent`, because is connecting directly to the document using `addListener`, not through `onClick`. I think the proper types is MouseEvent and the casting is needed because the MouseEvent target field is not exactly a Element, is an EventTarget which is a subset of Element based on the event type1